tests/tools/app_store_connect/conftest.py

```diff
@@ -42,3 +43,27 @@ def namespace_kwargs(mock_auth_key):
             continue
         os.environ.pop(arg.type.environment_variable_key, None)
     return ns_kwargs
+
+
+@pytest.fixture()
+def app_store_connect(namespace_kwargs) -> AppStoreConnect:
+    args = AppStoreConnectArgument
+    if "TEST_APPLE_PRIVATE_KEY_PATH" in os.environ:
+        key_path = pathlib.Path(os.environ["TEST_APPLE_PRIVATE_KEY_PATH"])
+        private_key = key_path.expanduser().read_text()
+        key_identifier = os.environ["TEST_APPLE_KEY_IDENTIFIER"]
+        issuer_id = os.environ["TEST_APPLE_ISSUER_ID"]
+    elif "TEST_APPLE_PRIVATE_KEY_CONTENT" in os.environ:
+        private_key = os.environ["TEST_APPLE_PRIVATE_KEY_CONTENT"]
+        key_identifier = os.environ["TEST_APPLE_KEY_IDENTIFIER"]
+        issuer_id = os.environ["TEST_APPLE_ISSUER_ID"]
+    else:
+        raise RuntimeError("Missing App Store Connect authentication information")
+
+    ns = namespace_kwargs | {
+        args.ISSUER_ID.key: Types.IssuerIdArgument(issuer_id),
+        args.KEY_IDENTIFIER.key: Types.KeyIdentifierArgument(key_identifier),
+        args.PRIVATE_KEY.key: Types.PrivateKeyArgument(private_key),
+    }
+    cli_args = argparse.Namespace(**ns)
+    return AppStoreConnect.from_cli_args(cli_args)
```

Comment:
This fixture doesn't seem to be used anywhere in the codebase, can you elaborate what purpose it serves?

Reply:
You are quite right that as of now it is not used anywhere. But it is very useful for quickly testing out actual command invocations using tests instead of doing it from terminal with
```shell
python -m codemagic.toos.app_store_connect <action> <args> ...
```
I could add an example test which uses this fixture, but is skipped unless live API tests are enabled. WDYT?

Reply:
I think let's add an example

Reply:
Added in [64ae629](https://github.com/codemagic-ci-cd/cli-tools/pull/383/commits/64ae62994dce0c92c5fc818a1f4fad1f6d5920dd).